core-server/src/main/java/org/glassfish/jersey/server/spi/Server.java

```diff
@@ -0,0 +1,13 @@
+package org.glassfish.jersey.server.spi;
+
+import java.util.concurrent.CompletionStage;
+
+public interface Server {
```

Comment:
I really miss the start() method here. I know that some servers do not have start, but the others do, and are started immediatelly. My idea is that the start could be optionally left on start() based on `JAXRS.Configuration start=false` property.

Reply:
Good point. But in case a backend cannot create a non-started server this would imply that Jersey simply creates a "fake" server instance. That is ok for me, if it is ok for you. :-)

Reply:
Not sure what you mean. If the backend cannot create a non-started server the start() method would throw an Unsupported Operation Exception...

Reply:
I will work on non-started containers this week.

Reply:
Non-started servers done. I refactored the factories a bit, so now you can even boot unstarted instances of _all_ backend types (even Netty and Simple which previously could only boot started).